go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/aws/aws-sdk-go v1.38.19
 	github.com/coreos/go-systemd/v22 v22.3.2
-	github.com/gardener/etcd-druid v0.8.0
-	github.com/gardener/gardener v1.48.0
+	github.com/gardener/etcd-druid v0.9.0
+	github.com/gardener/gardener v1.50.1-0.20220701074220-216e074e49da
```

Comment:
https://github.com/gardener/gardener/pull/6059/commits/ccaea580cfb6236a1a11ef7fc164f1b37b60697d makes the metrics bind address configurable. We can expose the metricsPort as a ControllerDeployment value in a similar way like it is done in this commit for provider-local. But if you want let's better do it in a separate follow-up PR.

Reply:
Addressed in follow-up PR #570.